### ISSUE TITLE
Does this work on Unified Interface V9?

### ISSUE THREAD

**Issue by andrewsun845** [PaulNieuwelaar/notifyjs#6]
We are currently trying to create the Unified Interface (APP) version of our D365 Online Instance, but when we are calling Notify.add() the javascript will always return the following:
Notify: CRM header element: 'undefined' does not exist.

Is V9 Unified Interface Supported for Notify JS?
If yes, is there any modification we need to do to call Notify.js

**Comment by PaulNieuwelaar** [comment]
Offically, no, v9/UUI is not yet supported for Notify.js in its current state. However I have been working on a new version which is currently working (somewhat) in UUI. It's not fully ready for a public release yet, as there's still a few layout issues, and the hacky code to get it to show up is not great... but all the functionality is there (and a few new color options too).

Feel free to try this version for now:

[NotifyJS_2_0_0_0.zip](https://github.com/PaulNieuwelaar/notifyjs/files/2618032/NotifyJS_2_0_0_0.zip)

**Comment by andrewsun845** [comment]
Awesome, the beta 2.0 Version seems to be working fine in UUI.
Looking forward to the formal release!

**Comment by RA-Danny** [comment]
Hi,
Do you have news regarding the v9/UUI release?

Many Thanks :)

**Comment by Greggomatic** [comment]
I was able to get v9 in an Apps view support working by looking at the mainContent tag

`Notify._crmContainerId = "mainContent"; //For Dynamic Apps Support`

And then validating its existence if all else fails

```
            // For Dynamics365 Apps, the container has a different name.
            if ($header.length == 0) {
                $header = $("#" + Notify._crmContainerId);
                if ($header.length > 0) {
                    $header.prepend($notify);
                }
            }	

```
Seems to work in all manners of usage.

Cheers.

**Comment by Mehran16007** [comment]
HI, did this problem solved? i have the same problem with UUI v9.1 ? how to solve it ? i install notify version 2.0 but it didn't help.
